src/test/functional/udf.spec.ts

```diff
@@ -1,154 +1,167 @@
-                    TestHelpers.getCollectionLink(isNameBased, db, collection),
-                    udfDefinition, undefined, client, isUpsertTest);
-                for (const property in udfDefinition) {
-                    if (property !== "serverScript") {
-                        assert.equal(udf[property], udfDefinition[property], "property " + property + " should match");
-                    } else {
-                        assert.equal(udf.body, "function () { const x = 10; }");
-                    }
-                }
 
-                // read udfs after creation
-                const { result: udfsAfterCreate } = await client.readUserDefinedFunctions(
-                    TestHelpers.getCollectionLink(isNameBased, db, collection)).toArray();
-                assert.equal(udfsAfterCreate.length, beforeCreateUdfsCount + 1, "create should increase the number of udfs");
-
-                // query udfs
-                const querySpec = {
-                    query: "SELECT * FROM root r WHERE r.id=@id",
-                    parameters: [
-                        {
-                            name: "@id",
-                            value: udfDefinition.id,
-                        },
-                    ],
-                };
-                const { result: results } = await client.queryUserDefinedFunctions(
-                    TestHelpers.getCollectionLink(isNameBased, db, collection), querySpec).toArray();
-                assert(results.length > 0, "number of results for the query should be > 0");
-
-                // replace udf
-                udf.body = function () { const x = 20; };
-                const { result: replacedUdf } = await TestHelpers.replaceOrUpsertUserDefinedFunction(
-                    TestHelpers.getCollectionLink(isNameBased, db, collection),
-                    TestHelpers.getUserDefinedFunctionLink(isNameBased, db, collection, udf),
-                    udf, undefined, client, isUpsertTest);
-
-                for (const property in udfDefinition) {
-                    if (property !== "serverScript") {
-                        assert.equal(replacedUdf[property], udf[property], "property " + property + " should match");
-                    } else {
-                        assert.equal(replacedUdf.body, "function () { const x = 20; }");
-                    }
+    describe("User Defined Function", function() {
+        it("nativeApi Should do UDF CRUD operations successfully", async function() {
+            const client = new CosmosClient({
+                endpoint,
+                auth: { masterKey }
+            });
+
+            // create database
+            await client.databases.create({
+                id: dbId
+            });
+
+            // create container
+            await client.databases
+                .getDatabase(dbId)
+                .containers.create({ id: "sample container" });
+
+            // read udfs
+            const container = await client.databases
```

Comment:
By keeping the `container` reference around, seems like `TestHelper` becomes unnecessary. At least for this test.